OpenRA.Mods.Common/Traits/Render/WithRepairOverlay.cs

```diff
@@ -36,19 +42,20 @@ public class WithRepairOverlayInfo : PausableConditionalTraitInfo, Requires<Rend
 	public class WithRepairOverlay : PausableConditionalTrait<WithRepairOverlayInfo>, INotifyDamageStateChanged, INotifyBuildComplete, INotifySold, INotifyRepair
 	{
 		readonly Animation overlay;
+		readonly WithRepairOverlayInfo info;
```

Comment:
`PausableConditionalTrait` already exposes an `Info` field, so this is redundant.